app/controllers/tournaments_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TournamentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def saveTournamentResults
+    #byebug
+    
+    #TODO: adding validation to the models
+    tournament = Tournament.new
```

Comment:
You can also use this syntax:
tournament = Tournament.create(tournament_id: params[:tournamentId], start_date: params[:startDate], ...)